wordpress-mobile/WordPress-iOS
# Add WordPress - Jetpack logo animation

Fixes #NA

This PR adds the WordPress - Jetpack logo animation to the "Jetpack powered" overlay

iPhone light | iPad dark
-- | --
<img height=350 src=https://user-images.githubusercontent.com/34376330/182735870-e2f40344-9630-47a6-b025-7e390bc57cf7.gif> | <img height=350 src=https://user-images.githubusercontent.com/34376330/182736103-86d8ccf4-b850-438c-ab1f-721cfc7e7764.gif>


**Note: this PR is only about animation: layout, fonts or other changes are not part of it.**

To test:
- Checkout this branch
- Enable the `jetpackPoweredBottomSheet` feature flag
- Build/run the branch
- Go to My Site / Home
- Tap on the "Jetpack powered" badge
- Make sure the overlay appears and the animation plays ad expected (see above)

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
